config.js

```diff
@@ -0,0 +1,24 @@
+import dotenv from "dotenv";
+
+dotenv.config()
+
+const config= {
+    development: {
+        port: process.env.PORT,
+        DB: process.env.DEVDB,
+        JWT_SECRET: process.env.JWT_SECRET
+    },
+    test: {
+        port: process.env.TEST_PORT,
+        DB: process.env.TESTDB,
+        JWT_SECRET: process.env.JWT_SECRET
+    },
+    production: {
+        port: process.env.PORT,
+        DB: process.env.DEVDB,
+        JWT_SECRET: process.env.JWT_SECRET
+    },
```

Comment:
For DB in production you need to make DATABASE_URL as environment variable. Because it will be used in production instead of DEVDB.